ui.R

```diff
@@ -172,6 +180,86 @@ shinyUI(fluidPage(
+                      span(title = "If someone could earn $1000 by working but would lose $400 in benefits and pay $200 in taxes, their participation tax rate would be 60% (losing $600 compared to a gross gain of $1000). A higher participation tax rate means people keep less of their earnings when they start working, which might discourage them from entering the workforce. This measure helps policymakers understand how tax and transfer systems together affect people's incentives to work versus remain on benefits.",
+                           class = "tooltip-span",
+                           "Example"),
+                      "$$PTR=1-\\frac{\\text{income in the hand from working}-\\text{income in the hand when not working}}{\\text{gross earnings from working}}$$"),
+              tags$li(strong("Income Composition"), " – The Income Composition tab shows how  earnings, taxes, and various government welfare transfers combine to give an individual’s net income.")
+            ),
+            h3("The following taxes and transfers are modelled in the Income Explorer:"),
+            tags$ul(
+              tags$li("Personal Income Tax (PIT)"),
+              tags$li("ACC Earners' Levy"),
+              tags$li("Working for Families (WFF):"),
+              tags$ul(
+                tags$li("Family Tax Credit (FTC)"),
+                tags$li("In-Work Tax Credit (IWTC)"),
+                tags$li("Minimum Family Tax Credit (MFTC)"),
+                tags$li("Best Start Tax Credit (BSTC)"),
+              ),
+              tags$li("Independent Earners Tax Credit (IETC)"),
+              tags$li("Core Benefits:"),
+              tags$ul(
+                tags$li("Jobseeker Support (JSS)"),
+                tags$li("Sole Parent Support (SPS)"),
+              ),
+              tags$li("Winter Energy Payment (WEP)"),
+              tags$li("Accommodation Supplement (AS)"),
+            ),
+            h3("Limitations of the Income Explorer"),
+            tags$ul(
+              tags$li(strong("Accommodation Supplement (AS)"), " – This tool assumes full
+                      take up of AS for all individuals with qualifying incomes and accommodation costs.
+                      However, AS eligibility is subject to additional tests, including asset tests, that are not currently included in the Income Explorer tool. To model an
+                      individual that is ineligible for/chooses not to take up AS, set their accommodation costs to zero."),
+              tags$li(strong("Jobseeker Support (JSS)"), " – This tool applies the ", em("Single, 25 years or over"), " JSS rate to 
+              all qualifying single people without children. It does not account for the different rates of JSS that are 
+              provided to people aged between 18 and 24 years older.")
+            ),
+            h3("The Income Explorer does not currently model:"),

```

Comment:
We don't model Kiwisaver contributions (personal or employer) either.

Reply:
Added